styled-switch-view/src/main/res/values/attrs_styled_switch.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <declare-styleable
+        name="StyledSwitch">
```

Comment:
Update the attribute name as the name is updated.

Reply:
Oh, just to name it as `StyledSwitchView`. It's quite minor, up to you. :)

Reply:
Sure thing!